quotes.js

```diff
@@ -0,0 +1,20 @@
+const quotes = document.getElementById("quote");
+const author = document.getElementById("author");
+const tags = document.getElementById("tag-item");
+
+async function getQuotes() {
+  const result = await fetch("https://api.quotable.io/random");
+  const resultJson = await result.json();
+
+  const content = resultJson.content;
+  const writer = resultJson.author;
+  const tagsArr = resultJson.tags;
+
+  quotes.innerHTML = content;
+  author.innerHTML = writer;
+  tagsArr.forEach(tag => {
+    tags.innerHTML = tag;
+  });
```

Comment:
if you have multiple tags in the response. It always shows the last tag.

Reply:
you have to build the the whole `<li id="tag-item" class="tag-item"></li>` inside the loop and after the loop you should change the inner html of `ul`.